### ISSUE TITLE
Nested pseudo selector within media queries don't work

### ISSUE THREAD

**Issue by jonaskuske** [morishitter/picostyle#36]
```javascript
const Paragraph = ps('p')({
  '@media screen and (min-width: 1000px)': {
    '::before': {
      content: '"Your screen is big!"'
    }
  },
  '::before': {
    content: '"Your screen is small!"'
  }
})
```

Using style objects within @media queries doesn't work correctly, the styles within the media queries are parsed correctly, but then not applied as part of the query but as global styles instead. So in the example above, the described Paragraph will always show *'Your screen is big!'*, no matter if the media query actually applies or not.
You can also checkout [this CodePen](https://codepen.io/21stCenturyJonas/pen/dddbNG?editors=0011), where only the title's color changes as part of the media query, not the title's pseudo-element's content. :)

**Comment by morishitter** [comment]
@jonaskuske I understand. Thank you for your report :)

**Comment by jonaskuske** [comment]
You're welcome, thanks for working on this library! :)
And just for accurate documentation, this is not just about pseudo selectors, combinators or anything else that works with nested style objects doesn't work either.  

For example
```javascript
const List = ps('ul')({
  '@media screen and (min-width: 600px)': {
    ' > li' : {
      listStyle: 'none' // always applied, ignores @media
    }
  }
});
```

**Comment by morishitter** [comment]
Fixed in #41
